code/modules/mob/living/simple_animal/friendly/cockroach.dm

```diff
@@ -0,0 +1,52 @@
+
+
+/mob/living/simple_animal/cockroach
+	name = "cockroach"
+	desc = "EWWWWWWWWWWWWWWWWWWWWW!"
+	icon_state = "cockroach"
+	icon_dead = "cockroach"
+	health = 1
+	maxHealth = 1
+	turns_per_move = 5
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 270
+	maxbodytemp = INFINITY
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	mob_size = MOB_SIZE_TINY
+	response_help  = "pokes"
+	response_disarm = "shoos"
+	response_harm   = "splats"
+	density = 0
+	ventcrawler = 2
+	gold_core_spawnable = 2
+	var/squish_chance = 50
+
+/mob/living/simple_animal/cockroach/death(gibbed)
+	if(ticker.cinematic) //If the nuke is going off, then cockroaches are invincible. Keeps the nuke from killing them, cause cockroaches are immune to nukes.
```

Comment:
Yes, this works. More of the meme that you cant nuke a cockroach since round ends 90 seconds after the nuke anyways.